Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/Migrations/LocalStorageContextModelSnapshot.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+namespace MvvmSeed.Domain.Model.Migrations
+{
+    [DbContext(typeof(LocalStorageContext))]
+    partial class LocalStorageContextModelSnapshot : ModelSnapshot
```

Comment:
This seems to be the solution to several problems that we used to have with EF:

- The need for a DB to actually create a new migration (even before coding it), because EF went to the DB to check where it was to automatically add pending changes to your to-be-created migration. This also means that you can have a single DB for your whole team instead of each local database.
- The need for those ugly resource files for each migration (no wonder why your approach here looks cleaner!)

This article explains it a lot longer, but a lot better: https://msdn.microsoft.com/en-us/library/dn481501(v=vs.113).aspx